src/MoBi.R/Services/SimulationFactory.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using MoBi.Core.Domain.Model;
+using MoBi.Core.Services;
+using MoBi.R.Domain;
+using OSPSuite.R.Domain;
+using OSPSuite.Utility.Extensions;
+using ModuleConfiguration = OSPSuite.Core.Domain.ModuleConfiguration;
+
+namespace MoBi.R.Services
+{
+   public interface ISimulationFactory
+   {
+      Simulation CreateSimulation(SimulationConfiguration configuration);
+   }
+
+   public class SimulationFactory : ISimulationFactory
+   {
+      private readonly ISimulationConfigurationFactory _configurationFactory;
+      private readonly Core.Domain.Services.ISimulationFactory _simulationFactory;
+
+      public SimulationFactory(ISimulationConfigurationFactory configurationFactory, Core.Domain.Services.ISimulationFactory simulationFactory)
+      {
+         _configurationFactory = configurationFactory;
+         _simulationFactory = simulationFactory;
+      }
+
+      public Simulation CreateSimulation(SimulationConfiguration configuration)
+      {
+         if (string.IsNullOrWhiteSpace(configuration.SimulationName))
+            throw new InvalidOperationException("Simulation name is required");
```

Comment:
Are the checks that the simulation name does not contain illegal characters and is not one of the forbidden names done elsewhere (e.g. when setting `configuration.SimulationName`) or should they be performed here?

compare 
https://github.com/Open-Systems-Pharmacology/MoBi/blob/472737a20c7514e99fd44cb3b6957c1de59e9844/src/MoBi.Presentation/DTO/ObjectBaseDTO.cs#L89
and
https://github.com/Open-Systems-Pharmacology/MoBi/blob/472737a20c7514e99fd44cb3b6957c1de59e9844/src/MoBi.Core/Services/ForbiddenNamesRetriever.cs#L35